libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/DocumentPermissions.java

```diff
@@ -78,6 +79,8 @@ protected boolean getStandardPermissions(RequestedAction action) {
             case USERS:
             case CLEARING:
                 return PermissionUtils.isAdmin(user) || isModerator();
+            case WRITE_ECC:
+                return PermissionUtils.isAdmin(user);
```

Comment:
WRITE_ECC is only applicable to releases. Normally, I'd want to deny everyone WRITE_ECC permission for any object that is not a release, but just to prevent possible future problems, this line ensures that ADMIN may do anything.